custom_components/tahoma/number.py

```diff
@@ -28,6 +28,36 @@
         max_value=4,
         entity_category=EntityCategory.CONFIG,
     ),
+    OverkizNumberDescription(
+        key=OverkizState.CORE_ECO_ROOM_TEMPERATURE,
+        name="Eco temperature",
```

Comment:
If we follow what we have done for the other numbers, it must be `Eco Room Temperature`.

@iMicknl what do you think?

Reply:
I would like that better I think, or if it doesn't make sense just Eco Temperature (capitalized).